homework_4/main.py

```diff
@@ -0,0 +1,78 @@
+from flask import Flask
+from webargs import fields
+from webargs.flaskparser import use_args
+
+from homework_4.utilities.utilities import format_records, execute_querry
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def hello_world():
+    return "<p>Hello, World!</p>"
+
+
+args_parser = {
+    "country": fields.Str(missing="all"),
+    "track_id": fields.Str(missing="1")
+}
+
+
+@app.route("/order-price")
+@use_args(args_parser, location="query")
```

Comment:
use validation in webargs lib. Not in code.

country = args_parser['country'] - this part you can use in webargs